code/atom.dm

```diff
@@ -704,6 +704,8 @@
 
 //mbc : sorry, i added a 'is_special' arg to this proc to avoid race conditions.
 /atom/proc/attackby(obj/item/W as obj, mob/user as mob, params, is_special = 0)
+	if((SEND_SIGNAL(src, COMSIG_ATTACKBY, W, user, params, is_special)) & ATTACK_PRE_DONT_ATTACK)
```

Comment:
Wait hold on, this probably shouldn't be here? At least not in this PR. Or if in this PR it probably shouldn't use the flag labeled as ATTACK_PRE...

Reply:
Forgot I left that in there. It was when I was trying to see i-- wait shit I forgot if I actually used that or not. One of the components does listen for COMSIG_ATTACKBY. Uh. I removed the che-- wait, no I was right the first time, I don't use that anymore. Removed!